crates/cli/src/bin/wasm-bindgen-test-runner/main.rs

```diff
@@ -48,7 +48,7 @@ fn main() {
         Err(e) => e,
     };
     eprintln!("error: {}", err);
-    for cause in err.causes().skip(1) {
```

Comment:
Yay! I've literally had this `skip(1)` in every single project that used `failure`!